webapp/src/components/expanded_view/component.tsx

```diff
@@ -1027,13 +1040,16 @@ export default class ExpandedView extends React.PureComponent<Props, State> {
             });
         }
 
+        const liveCaptionsText = this.state.showLiveCaptions ? formatMessage({defaultMessage: 'Turn off live captions'}) : formatMessage({defaultMessage: 'Turn on live captions'});
+
         const globalRhsSupported = Boolean(this.props.selectRhsPost);
 
         const isChatUnread = Boolean(this.props.threadUnreadReplies);
 
         const isHost = this.props.callHostID === this.props.currentUserID;
 
         const isRecording = isHost && this.props.isRecording;
+        const showCCButton = this.props.transcriptionsEnabled && this.props.isRecording && Boolean(this.props.callRecording?.start_at);
```

Comment:
Adding a comment here since I noticed it on Community: we should hide if live captions are disabled.

Reply:
Thanks, fixed in [3b22c07](https://github.com/mattermost/mattermost-plugin-calls/pull/633/commits/3b22c075324ebeb5777cc81e736252fcb34061e3)